src/repo.rs

```diff
@@ -343,6 +360,10 @@ impl DocState {
         matches!(self, DocState::Sync(_))
     }
 
+    fn should_request(&self) -> bool {
```

Comment:
There is `is_bootstrapping` above that does the same thing, pick one?

Reply:
Done